hypershift-operator/controllers/hostedcluster/hostedcluster_controller.go

```diff
@@ -216,19 +208,13 @@ func (r *HostedClusterReconciler) managedResources() []client.Object {
 		&rbacv1.ClusterRoleBinding{},
 		&rbacv1.Role{},
 		&rbacv1.RoleBinding{},
-		&corev1.ConfigMap{},
 		&corev1.Secret{},
 		&corev1.Namespace{},
 		&corev1.ServiceAccount{},
-		&corev1.Service{},
 		&agentv1.AgentCluster{},
 		&capiibmv1.IBMVPCCluster{},
 		&capikubevirt.KubevirtCluster{},
 	}
-	// Watch based on Routes capability
```

Comment:
why are we removing the CapabilityRoute check?

Reply:
The unit test `TestHostedClusterWatchesEverythingItCreates` detected that the HO no longer creates Routes and thus doesn't need to watch them (along with ConfigMaps and Services).  The point of this change is to prevent Routes from being added to the list of `managedResources`.